server/server.go

```diff
@@ -0,0 +1,168 @@
+package server
+
+import (
+	"fmt"
+	"net"
+	"context"
+	"strings"
+	"romano.com/proto"
+	"romano.com/mongo"
+	"romano.com/model/fileModel"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Server struct {
+	port string
```

Comment:
You should add the `*mongo.Client` you create as a field in this struct after removing `mongo.Controller` since it has no use besides holding a `*mongo.Client` as I commented on the mongo/controller.go file.